packages/mobile/src/invite/utils.ts

```diff
@@ -38,3 +41,31 @@ export function extractValidInviteCode(inviteFieldInput: string) {
     return inviteCode
   }
 }
+
+interface ReferrerData {
+  clickTimestamp: string
+  installReferrer: string
+  installTimestamp: string
+}
+
+interface ReferrerDataError {
+  message: string
+}
+
+export const getInviteCodeFromReferrerData = async () => {
+  const referrerData: ReferrerData | ReferrerDataError = await RNInstallReferrer.getReferrer()
+  Logger.info(
+    'invite/utils/getInviteCodeFromReferrerData',
+    'Referrer Data: ' + JSON.stringify(referrerData)
+  )
+  if (referrerData && referrerData.hasOwnProperty('installReferrer')) {
```

Comment:
What are your thoughts on `hasOwnProperty`?
I used to use it exclusively but since joining Celo I've been just using `referrerData.installReferrer ` instead.